proto/viam/app/v1/app.proto

```diff
@@ -371,8 +371,10 @@ message DeleteOrganizationMemberResponse {}
 message LocationOrganization {
   // Organization ID the location is shared with.
   string organization_id = 1;
+  // Organiztion name the location is shared with.
+  string organization_name = 2;
```

Comment:
I don't quite understand why this field is required. If i'm not mistaken, the [app PR](https://github.com/viamrobotics/app/pull/2780/files#diff-515b343b0aed1bc7f56fc1d71a890e243bc0e3f154c7edd8d101193997083a7c) doesn't use this updated copy of `api` yet so I wasn't able to find your utilization of the field. Let me know if there's somewhere else I should be looking!